test/integration/setup/adminclient/build.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+set -e
+
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+
+cd "${DIR}" && GOOS=linux go build -o $1
```

Comment:
Can you please add CGO_ENABLED=0 to this? It makes it easier to do development since you can compile outside Docker and run inside Docker (alpine)